tests/runtimes/local/app/oxo_test.py

```diff
@@ -52,3 +52,274 @@ def testImportScanMutation_always_shouldImportScan(
+    with models.Database() as session:
+        scans = session.query(models.Scan).filter(models.Scan.id.in_([1, 2])).all()
+        assert scans is not None
+
+    query = """
+        query Scans($scanIds: [Int!]) {
+            scans(scanIds: $scanIds) {
+                scans {
+                    id
+                    title
+                    asset
+                    progress
+                    createdTime
+                }
+            }
+        }
+    """
+
+    response = client.post(
+        "/graphql", json={"query": query, "variables": {"scanIds": [1, 2]}}
+    )
+
+    assert response.status_code == 200
+    response_json = response.get_json()
+    assert response_json["data"]["scans"]["scans"][0]["id"] == 2
+    assert response_json["data"]["scans"]["scans"][0]["title"] == scans[1].title
+    assert response_json["data"]["scans"]["scans"][0]["asset"] == scans[1].asset
+    assert response_json["data"]["scans"]["scans"][0]["progress"] == str(
+        scans[1].progress
+    )
+    assert (
+        response_json["data"]["scans"]["scans"][0]["createdTime"]
+        == scans[1].created_time.isoformat()
+    )
+    assert response_json["data"]["scans"]["scans"][1]["id"] == 1
+    assert response_json["data"]["scans"]["scans"][1]["title"] == scans[0].title
+    assert response_json["data"]["scans"]["scans"][1]["asset"] == scans[0].asset
+    assert response_json["data"]["scans"]["scans"][1]["progress"] == str(
+        scans[0].progress
+    )
+    assert (
+        response_json["data"]["scans"]["scans"][1]["createdTime"]
+        == scans[0].created_time.isoformat()
+    )
+
+
+def testQueryMultipleVulnerabilities_always_shouldReturnMultipleVulnerabilities(
+    client: flask.testing.FlaskClient, ios_scans: models.Scan
+) -> None:
+    """Test query for multiple vulnerabilities."""
+    with models.Database() as session:
+        vulnerabilities = (
+            session.query(models.Vulnerability)
+            .filter(models.Vulnerability.id.in_([1, 2]))
+            .all()
+        )
+        assert vulnerabilities is not None
+
+    query = """
+        query Vulnerabilities($vulnerabilityIds: [Int!]) {
+            vulnerabilities(vulnerabilityIds: $vulnerabilityIds) {
+                vulnerabilities {
+                    id
+                    technicalDetail
+                    riskRating
+                    cvssV3Vector
+                    dna
+                    title
+                    shortDescription
+                    description
+                    recommendation
+                    references
+                    location
+                }
+            }
+        }
+    """
+
+    response = client.post(
+        "/graphql", json={"query": query, "variables": {"vulnerabilityIds": [1, 2]}}
+    )
+
+    assert response.status_code == 200
+    response_json = response.get_json()
+    assert response_json["data"]["vulnerabilities"]["vulnerabilities"][0]["id"] == 2
```

Comment:
put `response_json["data"]["vulnerabilities"]["vulnerabilities"]`(or even the individual vulnz) in variables.

Reply:
+1